packages/dds/map/src/interfaces.ts

```diff
@@ -301,10 +302,12 @@ export interface IDirectoryValueChanged extends IValueChanged {
 
 /**
  * Events emitted in response to changes to the {@link ISharedMap | map} data.
+ * @privateRemarks
+ * Does not include ISharedObjectEvents since thats not part of the declarative/public API surface.
  * @sealed
- * @alpha
+ * @beta
  */
-export interface ISharedMapEvents extends ISharedObjectEvents {
+export interface ISharedMapEvents extends IErrorEvent {
```

Comment:
This is significant, and may break someone. Note that the existing ISharedMap still includes ISharedObjectEvents, so this only breaks explicit use of ISharedMapEvents.